Core/models.py

```diff
@@ -104,6 +103,8 @@ class Member(models.Model):
     github = models.CharField(max_length=225, null=True, default=None)
     linkedin = models.CharField(max_length=225, null=True, default=None)
 
+    # message = models.TextField(null=True, blank=True)
```

Comment:
uncomment this

Reply:
@bvishal8510 sir it is the code for alumni section and as designs are not ready, I have commented the entire code for it

Reply:
but this is not according to PEP8

Reply:
So remove it. A PR should only contain relevant changes.